Python/omevv/v1/scripts/GetRepoDetails.py

```diff
@@ -0,0 +1,196 @@
+        self.repo_url = "https://%s/Spectre/api/rest/v1/Services/PluginProfileService/RepositoryProfiles"%(omevv_ip)
+        self.omevv_encoded_cred = self.encode_cred(self.omevv_username, self.omevv_password)
+        self.payload_cred_dict= {}
+        self.payload_cred_dict['console'] = 'consoleUserCredential'
+        self.payload_cred_dict['api'] = 'apiUserCredential'
+        self.retry = 3
+
+
+    def encode_cred(self,username, pwd):
+        cred_str = username + ":" + pwd;
+        cred_str_bytes = cred_str.encode("ascii");
+        base64_bytes = base64.b64encode(cred_str_bytes);
+        base64_string = base64_bytes.decode("ascii");
+        return base64_string
+
+    def create_payload(self,username,pwd,domain,type):
+        payload = {}
+        cred_json = {"username": username.strip(), "domain": domain.strip(), "password": pwd.strip()};
+        payload[self.payload_cred_dict[type]] = cred_json
+        return payload
+
+    def create_context_payload(self,cid,console_username,console_domain,console_pwd):
+        payload = {}
+        payload['consoleId'] = cid
+        cred_json = {"username": console_username.strip(), "domain": console_domain.strip(), "password": console_pwd.strip()};
+        payload["consoleUserCredential"] = cred_json
+        return payload
+
+
+    def create_login(self):
+       bearer_token = ""
+       print(self.omevv_encoded_cred)
+       self.headers['Authorization'] = self.omevv_encoded_cred
+       cred_json = self.create_payload(self.omevv_username,self.omevv_password,self.domain,"api")
+       print("cred_json ",cred_json)
+       response = requests.post(self.login_url, data=json.dumps(cred_json), headers=self.headers, verify=False);
+       data = response.json();
+       status_code = response.status_code
+       if status_code == 200:
+           print("Sucesfully logged in");
+           bearer_token = data["accessToken"];
+           self.bearer_token = bearer_token
+       else:
+           raise Exception("Error occured while logging in ", data);
+      # print(bearer_token)
+       return bearer_token
+
+    def get_console_details(self):
+        console_list = []
+
+        try:
+            bearer_token = self.create_login()
+            cred_str = 'Bearer ' + bearer_token
+            self.headers['Authorization'] = cred_str
+            print("headers ",self.headers)
+            response = requests.get(self.console_url, headers=self.headers, verify=False)
+            data = response.json();
+            status_code = response.status_code;
+            print("status code in console ",status_code)
+            if status_code == 200:
+                for con in data:
+                    s = json.dumps(con)
+                    conObj = json.loads(s, object_hook=consoleDecoder)
+                    console_list.append(conObj)
+            else:
+                raise Exception("Error Occured while fetching the repo profile details ", response)
```

Comment:
Does not seem like the right representation of the error. Can you add a more appropriate error message?